rust_src/src/numbers.rs

```diff
@@ -1,6 +1,7 @@
 //! Functions operating on numbers.
 
 use rand::{Rng, SeedableRng, StdRng};
```

Comment:
While you are here, can you move the `rand` import to its own line after the `std` imports?

Reply:
Sure thing.